src/components/Books.js

```diff
@@ -30,20 +30,48 @@ const useGetBook = (bookId) => {
     };
 };
 
-const updateBookStatus = async (bookId, checkedOutBy) => {
-    return await documentClient.update({
+const updateBookToReturned = (bookId, userProfile) => {
+    const event = {
+        timestamp: new Date(Date.now()).toJSON(),
```

Comment:
What is the difference between `new Date(Date.now())` and `new Date()`? I didn't notice a difference, but I was testing in the browser and don't see a harm in having it.

Reply:
No difference apparently. but it's probably more clear for those who aren't JS Date experts